jerry-core/ecma/builtin-objects/ecma-builtin-array.inc.h

```diff
@@ -62,4 +62,6 @@ ROUTINE (LIT_MAGIC_STRING_IS_ARRAY_UL, ecma_builtin_array_object_is_array, 1, 1)
 #undef STRING_VALUE
 #undef OBJECT_VALUE
 #undef ROUTINE
-
+#undef ACCESSOR_VALUE
+#undef ACCESSOR_RO_VALUE
+#undef ACCESSOR_WO_VALUE
```

Comment:
Please remove ACCESSOR_RO_VALUE and ACCESSOR_WO_VALUE. Only keep ACCESSOR_VALUE.